docs/dev/run_the_etl.rst

```diff
@@ -200,13 +388,30 @@ years.
 Additional Notes
 ----------------
 The commands above should result in a bunch of Python :mod:`logging` output
-describing what the script is doing, and file outputs in the ``sqlite``,  and
-``parquet`` directories within your workspace. When the ETL is complete, you
-should see new files at ``sqlite/ferc1.sqlite`` and ``sqlite/pudl.sqlite`` as
-well as a new directory at ``parquet/epacems`` containing nested directories
-named by year and state.
-
-If you need to re-run ``ferc_to_sqlite`` or ``pudl_etl`` and want to overwrite
-their previous outputs you can add ``--clobber`` (run ``script_name --clobber``).
+describing what the script is doing, and file outputs in the ``outputs``
```

Comment:
The terminal asks for /output in the warning. It'd be helpful to standardize across all the new docs & code which one we're asking for to avoid confusion. This has also come up here: https://github.com/catalyst-cooperative/pudl/pull/2356#issuecomment-1454804964

Reply:
I'm gonna make an executive decision and use `output` here because I think which one Does Not Matter and we just need to pick one.

Reply:
We might want to make the environment variable and the recommended directory name the same or similar to link them in people's minds by default, like `PUDL_OUTPUT=`/path/to/your/pudl_output/

Reply:
You mean instead of `~/pudl-work/output`, we have `~/pudl_output`?

Also in https://github.com/catalyst-cooperative/pudl/pull/2401 I made everything `PUDL_OUTPUT` and `PUDL_INPUT` (instead of `PUDL_OUT[PUT]` and `PUDL_CACHE`) - would you prefer `PUDL_DATA` as the input env var name? Or perhaps replace `~/pudl_data` with `~/pudl_input`?

Reply:
I think `pudl_input` and `pudl_output` are good. "data" is so generic in all this. It's everywhere!